packager/generateModulePaths.js

```diff
@@ -21,7 +21,7 @@ const modulePaths = moduleNames.map((moduleName) => {
     }
     try {
         const result = execSync(
-            `grep "@providesModule ${moduleName}" $(find . -name ${moduleName}\\\\.js) -l`
+            `grep "@providesModule ${moduleName}" $(find . -name ${moduleName}\\\\.js) -l`,
```

Comment:
Why a dangling comma here?

Reply:
Eslint update

Reply:
Yeah... I thought it was odd for function call arguments rather than for objects, but the rule is for all types of multiline, so all good.